FIX: Support PyPy > 3.7 (#480)
cloudpipe/cloudpickle: CHANGES.md

2.2.0 (in development)
======================

> - Fix import of pickle module on PYPY 3.8.

Reviewer: ```suggestion
- Fix support of PyPy 3.8 and later.
```
